projects/nutanix-cloud-native/cluster-api-provider-nutanix/Makefile

```diff
@@ -0,0 +1,30 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
```

Comment:
not a blocker, but is there a plan on the capx side to upgrade to 1.18 or 1.19?

Reply:
We are planning on upgrading CAPX to 1.19; will ofc update that here once CAPX is upgraded.
cc @deepakm-ntnx